typescript/infra/scripts/check/check-utils.ts

```diff
@@ -63,7 +67,9 @@ export function getCheckWarpDeployArgs() {
 }
 
 export function getCheckDeployArgs() {
-  return withWarpRouteId(withModule(getCheckBaseArgs()));
+  return withGithubCommit(
+    withWarpRouteIdNoChoices(withModule(getCheckBaseArgs())),
+  );
 }
```

Comment:
this will break a lot of existing flows if we're now demanding that a github commit has to be required with every check-deploy call - imo it should still be optional

Reply:
it is optional though

Reply:
the change from withWarpRouteId -> withWarpRouteIdNoChoices has the implication that `warpRouteIds` are now demanded, as per this line:
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/faabe65cf579101d6d68a5badd6ba049848768cf/typescript/infra/scripts/agent-utils.ts#L213

which makes sense in check-warp-deploy but not in check-deploy

Reply:
Gotcha. removed!